src/VideoIntelligence/V1beta1/VideoIntelligenceServiceClient.php

```diff
@@ -0,0 +1,428 @@
+     *     $inputUri = "";
+     *     $features = [];
+     *     $operationResponse = $videoIntelligenceServiceClient->annotateVideo($inputUri, $features);
+     *     $operationResponse->pollUntilComplete();
+     *     if ($operationResponse->operationSucceeded()) {
+     *       $result = $operationResponse->getResult();
+     *       // doSomethingWith($result)
+     *     } else {
+     *       $error = $operationResponse->getError();
+     *       // handleError($error)
+     *     }
+     *
+     *     // OR start the operation, keep the operation name, and resume later
+     *     $operationResponse = $videoIntelligenceServiceClient->annotateVideo($inputUri, $features);
+     *     $operationName = $operationResponse->getName();
+     *     // ... do other work
+     *     $newOperationResponse = $videoIntelligenceServiceClient->resumeOperation($operationName, 'annotateVideo');
+     *     while (!$newOperationResponse->isDone()) {
+     *         // ... do other work
+     *         $newOperationResponse->reload();
+     *     }
+     *     if ($newOperationResponse->operationSucceeded()) {
+     *       $result = $newOperationResponse->getResult();
+     *       // doSomethingWith($result)
+     *     } else {
+     *       $error = $newOperationResponse->getError();
+     *       // handleError($error)
+     *     }
+     * } finally {
+     *     $videoIntelligenceServiceClient->close();
+     * }
+     * ```
+     *
+     * @param string    $inputUri     Input video location. Currently, only
+     *                                [Google Cloud Storage](https://cloud.google.com/storage/) URIs are
+     *                                supported, which must be specified in the following format:
+     *                                `gs://bucket-id/object-id` (other URI formats return
+     *                                [google.rpc.Code.INVALID_ARGUMENT][google.rpc.Code.INVALID_ARGUMENT]). For more information, see
+     *                                [Request URIs](/storage/docs/reference-uris).
+     *                                A video URI may include wildcards in `object-id`, and thus identify
+     *                                multiple videos. Supported wildcards: '&#42;' to match 0 or more characters;
+     *                                '?' to match 1 character. If unset, the input video should be embedded
+     *                                in the request as `input_content`. If set, `input_content` should be unset.
+     * @param Feature[] $features     Requested video annotation features.
+     * @param array     $optionalArgs {
+     *                                Optional.
+     *
+     *     @type string $inputContent
+     *          The video data bytes. Encoding: base64. If unset, the input video(s)
+     *          should be specified via `input_uri`. If set, `input_uri` should be unset.
+     *     @type VideoContext $videoContext
```

Comment:
We should add a reference to `VideoContext` in our [external references](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/docs/external-classes.json).